### PR TITLE
 Expose StorageAccountNameStep and StorageAccountCreateStep

### PR DESCRIPTION
**Pull request by PrashanthCorp**
Need this for creation of storage accounts in the storage extension. 
Reqd. for PR: https://github.com/Microsoft/vscode-azurestorage/pull/290

### CODE REVIEW

File: ui/package-lock.json [hunk 1]
```diff
@@ -50,26 +50,26 @@
...
-                "async": ">=0.6.0",
-                "date-utils": "*",
-                "jws": "3.x.x",
-                "request": ">= 2.52.0",
-                "underscore": ">= 1.3.1",
-                "uuid": "^3.1.0",
-                "xmldom": ">= 0.1.x",
-                "xpath.js": "~1.1.0"
+                "@types/node": "8.10.38",
+                "async": "2.6.0",
```
Comment (EricJizbaMSFT) [inline ui/package-lock.json]: Need to figure out why your npm is so different. I think @nturinski  is the last person to edit this file. Ask him what version he's on

Comment (nturinski) [reply ui/package-lock.json]: v6.5.0

Comment (PrashanthCorp) [reply ui/package-lock.json]: I pushed this from another machine, I'll have to update the npm version

**Comment by PrashanthCorp** [comment]
1. Could not find the `.DS_Store` file when I pulled this. Maybe a windows behavior. Given the use case of `.DS_Store`, I strongly feel we should add it to `.gitignore`

**Comment by EricJizbaMSFT** [comment]
`.DS_Store` is a file generated by the Mac OS. I'm not gonna complain if you add it to our gitignore, but I actually recommend you add a global gitignore that excludes it from all repos on your machine. See here for more info: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Pull request merged**